ddtrace/contrib/elasticsearch/transport.py

```diff
@@ -10,6 +11,21 @@
 SPAN_TYPE = 'elasticsearch'
 
 
+class TracedConnection(Urllib3HttpConnection):
+    """Change to elasticsearch http connector so that it
+        adds the http status code to data
+    """
+
+    def perform_request(self, method, url, params=None, body=None, timeout=None, ignore=()):
+        status, headers, data = super(TracedConnection, self).perform_request(method, url, params,
+            body, ignore=ignore, timeout=timeout)
+        import json
```

Comment:
follow pep8 for `import` style https://www.python.org/dev/peps/pep-0008/#imports